app/src/main/java/com/it/hungvt/movieapp/MainActivity.java

```diff
@@ -1,10 +1,12 @@
 package com.it.hungvt.movieapp;
 
+import android.app.FragmentManager;
```

Comment:
ERROR: Unused import: android.app.FragmentManager. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports